class.jetpack-xmlrpc-server.php

```diff
@@ -112,7 +112,14 @@ function remote_authorize( $request ) {
 		if ( is_wp_error( $result ) ) {
 			return $this->error( $result );
 		}
-
+		/**
+		 * Fires after the Jetpack client is authorized to communicate with WordPress.com.
+		 *
+		 * @since 5.3.0
+		 *
+		 * @param int Jetpack Blog ID.
+		 */
+		do_action( 'jetpack_client_authorized', Jetpack_Options::get_option( 'id' ) );
```

Comment:
This totally makes sense, but can't we use the `jetpack_user_authorized` action that is already being triggered inside `Jetpack_Client_Server` here?
https://github.com/Automattic/jetpack/blob/master/class.jetpack-client-server.php#L270

I like the idea of passing the ID as a parameter to the action, but we're essentially creating a duplicate here, would you agree?